src/AppBundle/Resources/translations/report-prof-deputy-costs.en.yml

```diff
@@ -0,0 +1,127 @@
+startPage:
+  htmlTitle: "Deputy report - Deputy costs | GOV.UK"
+  pageTitle: Deputy costs
+  pageSectionDescription1: |
+    Use this section to tell us how much %client% paid for your professional deputy services
+     in this reporting period (15 March 2017 to 14 March 2018).
```

Comment:
Date hard coded. Estimates has this method in place to calculate (see start page of estimates)

Reply:
Worth creating a Jira ticket in the sprint to prevent getting this deployed?